ocaml/xapi/xapi_pool.ml

```diff
@@ -1318,6 +1389,40 @@ let join_common ~__context ~master_address ~master_username ~master_password
   in
   finally
     (fun () ->
+      (* Merge certificates used for trusting appliances, also known as ca
+         certificates. At this point the names of certificates have been tested
+         for uniqueness across pools, the name of the certificate is used to
+         identify each certificate. *)
```

Comment:
It would make more sense to me to use the uuids here. Is it the case that we unfortunately have identical certs in two databases, with different uuids?

Reply:
Ideally we would use uuids instead of whatever name the client gave to the certificate. This is not the case unfortunately so we use those names instead.